tests/hazmat/backends/test_openssl.py

```diff
@@ -529,7 +534,7 @@ def test_checks_for_unsupported_extensions(self):
         ).not_valid_after(
             datetime.datetime(2020, 1, 1)
         ).add_extension(
-            x509.IssuerAlternativeName([x509.DNSName(u"crypto.io")]), False
+            DummyExtension(), False
```

Comment:
ah, ok

On Sun, Aug 9, 2015 at 11:58 AM, Paul Kehrer <notifications@github.com>
wrote:

> In tests/hazmat/backends/test_openssl.py
> <https://github.com/pyca/cryptography/pull/2232#discussion_r36589782>:
>
> > @@ -529,7 +534,7 @@ def test_checks_for_unsupported_extensions(self):
> >          ).not_valid_after(
> >              datetime.datetime(2020, 1, 1)
> >          ).add_extension(
> > -            x509.IssuerAlternativeName([x509.DNSName(u"crypto.io")]), False
> > +            DummyExtension(), False
>
> That test is for CSRBuilder. Removing this test results in an uncovered
> line (just checked locally). I'll move this test into test_x509 though
> since it's not really backend specific.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/pyca/cryptography/pull/2232/files#r36589782>.
>



-- 
"I disapprove of what you say, but I will defend to the death your right to
say it." -- Evelyn Beatrice Hall (summarizing Voltaire)
"The people's good is the highest law." -- Cicero
GPG Key fingerprint: 125F 5C67 DFE9 4084